src/main/java/com/kodilla/ecommercee/ProductController.java

```diff
@@ -1,4 +1,35 @@
 package com.kodilla.ecommercee;
 
+import com.kodilla.ecommercee.dto.ProductDTO;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collections;
+import java.util.List;
+
+@RequestMapping("v1/products")
+@RestController
+@CrossOrigin("*")
 public class ProductController {
-}
+    @GetMapping()
+    public ResponseEntity<List<ProductDTO>> getProducts() {
+        return ResponseEntity.ok(Collections.emptyList());
+    }
```

Comment:
Róbmy odstępy między metodami :)

Reply:
ok, ten PR jest już zamknięty. Dodam to jako poprawka do zadania JDP230301-5